### PR TITLE
fix: the type of rules and some variable names

### PR DESCRIPTION
**Pull request by dovics**
Signed-off-by: dovics <wrs369@163.com>

### CODE REVIEW

File: node.go [hunk 1]
```diff
@@ -147,6 +153,23 @@ func (n *Node) EnableTLSTransport(keyFile string, certFile string, caFile string
 	n.enableTLS = true
 }
 
+// EnableTLSTransportFromText make transport protected by TLS.
+// etcd.pkg.transport.TLSInfo don't support use context as the cert. so
+// this function will create the cert file
+func (n *Node) EnableTLSTransportFromText(key []byte, cert []byte, ca []byte) error {
+	if err := creatAndWriteFile(key, DefaultKeyFileName); err != nil {
```
Comment (nodece) [inline node.go]: You don't should create file for key, cert and ca, please consider the use `tls.X509KeyPair`.

Comment (dovics) [reply node.go]: The problem is transport only provides  set up TLS from file path, 
https://github.com/etcd-io/etcd/blob/master/pkg/transport/listener.go#L66

Comment (nodece) [reply node.go]: If etcd only supports set up TLS from file, it unnecessary that add `EnableTLSTransportFromText`.

**Review by nodece — APPROVED** [review]
LGTM

**Pull request merged**